docs/core/native-interop/expose-components-to-com.md

```diff
@@ -80,14 +80,24 @@ Open an elevated command prompt and run `regsvr32 ProjectName.comhost.dll`. That
 
 The resulting output will now also have a `ProjectName.X.manifest` file. This file is the side-by-side manifest for use with Registry-Free COM.
 
+## Embedding Type Libraries in the COM host
+
+Unlike in .NET Framework, there is no support in .NET Core or .NET 5 or newer for generating a COM Type Library (TLB) from a .NET Core assembly. The guidance is to either manually write an IDL file or a C/C++ header for the native declarations of the COM interfaces. If you decide to write an IDL file, you can compile it with the Visual C++ SDK's MIDL compiler to produce a TLB.
+
+In .NET 6 Preview 5 or newer, the .NET SDK supports embedding already-compiled TLBs into the COM host as part of your project build.
+
+To embed a type library into your application, follow these steps:
+
+1. Open the `.csproj` project file and add `<ComHostTypeLibrary Include="path/to/typelib.tlb" Id="<id>" />` inside an `<ItemGroup></ItemGroup>` tag.
```

Comment:
Might be nice to just have a code block with this instead of saying to put it in an ItemGroup.